guides/source/autoloading_and_reloading_constants.md

```diff
@@ -461,8 +461,9 @@ Also, this collection is configurable via `config.autoload_paths`. For example,
 by adding this to `config/application.rb`:
 
 ```ruby
-config.autoload_paths += "#{Rails.root}/lib"
+config.autoload_paths += ["#{Rails.root}/lib"]
```

Comment:
Changed that, thanks @egilburg! I amended and force pushed to keep it as one commit, I hope that's ok